integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -256,32 +257,39 @@ public void exec(int depth, int concurrencyDepth, TachyonURI path) throws Except
    */
   private static final String TEST_AUTHENTICATE_USER = "test-user";
 
-  private ExecutorService mExecutorService = null;
   @Rule
   public LocalTachyonClusterResource mLocalTachyonClusterResource =
       new LocalTachyonClusterResource(1000, 1000, Constants.GB,
           Constants.SECURITY_AUTHENTICATION_TYPE, AuthType.SIMPLE.getAuthName());
   private TachyonConf mMasterTachyonConf;
   private FileSystemMaster mFsMaster;
+  private User mOldUser;
 
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
 
   @After
   public final void after() throws Exception {
-    mExecutorService.shutdown();
+    if (mOldUser == null) {
+      AuthorizedClientUser.remove();
+    } else {
+      AuthorizedClientUser.set(mOldUser.getName());
```

Comment:
If the static field is unset at the start of FileSystemMasterIntegrationTest, `mOldUser` will always be null. However, both cases are handled to avoid depending on that assumption.